docs/getting-started/pipeline.yaml

```diff
@@ -94,15 +98,13 @@ spec:
         type: image
   steps:
     - name: build-and-push
-      image: r.j3ss.co/img
+      image: gcr.io/kaniko-project/executor:v0.16.0
       command:
-        - /usr/bin/img
+        - /kaniko/executor
```

Comment:
Do we need to do anything here to setup the docker registry creds for Kaniko? 
The pipeline tutorial has users setting up a secret and a service account: https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md#configuring-task-execution-credentials

Do we need to do the same?

Reply:
Good point. It might come down to the k8s instance and the overall setup we run in. I used GKE and it seemed to work with this yaml. Maybe my default user/SA has the full access or something by default and no need to do that. But if you do use another service account maybe we would need that.

Reply:
Yeah, I think that's why it works. We should point that out in a note with maybe a link to the section on setting up registry auth in the pipelines tutorial?

Reply:
Yup. I had a chat with @wlynch and he also mentioned on GKE without workload identity setup it goes through the default GCE that has the editor access and it just works. So I added a note to the readme to cover that case. Let me know if you think that is sufficient.

Reply:
Yup, just verified the docs: https://cloud.google.com/container-registry/docs/using-with-google-cloud-platform#gke-permissions